README.md

```diff
@@ -144,4 +144,13 @@ To track how this government-funded code is used, we request that if you decide
 
 Digital Object Identifier: (fill in with DOI)
 
+To cite this code in a publication or report, please cite our associated report/paper and/or our source code:
+(Fill out the `formatted fields` in the below citation with details of your source code)
+
+`author_surname_or_organization`, `initial`. (`year`). _`program_or_source_code_title`_ (`code_version`) [Source code]. Provided by ITS CodeHub through GitHub.com. Accessed YYYY-MM-DD from `doi_url`.
+
+(Also provide citation for companion paper publication, if available)
+
+To cite this code when you copy or adapt from this code, please include the original URL you copied from and date of retrieval as a comment in your code. Additional information on how to cite can be found in the [ITS CodeHub FAQ](https://its.dot.gov/code/#/faqs).
```

Comment:
which original URL? Original URL of the source code?

Reply:
Yes- original URL where one copied the source code from. This is following the guidance at https://integrity.mit.edu/handbook/writing-code and allows for more specific attribution than DOI.

Reply:
I suggest making slight change to wording - "original URL you copied the source code from..."